### ISSUE TITLE
cy.readFile() must be passed a non-empty string as its 1st argument.

### ISSUE THREAD

**Issue by Naethaniel** [abhinaba-ghosh/cypress-react-selector#120]
Hello,

Recently we have updated our cypress-related packages:

"@testing-library/cypress": "^6.0.0" => "@testing-library/cypress": "^7.0.3"
"cypress": "^5.4.0" => "cypress": "6.2.1"
"cypress-react-selector": "^2.2.0" => "cypress-react-selector": "2.2.2"

After those changes we are having problem with `cy.waitForReact();`
In our test we are calling this method inside `it()`. 
We are getting error:
`cy.readFile() must be passed a non-empty string as its 1st argument. You passed: 396.`
Before the update everything worked correctly.

cypress.json
```
{
    "baseUrl": "http://localhost:3000",
    "cypress-react-selector": {
        "root": "selector"
    }
}
```

tsconfig.json
```
{
    "extends": "../tsconfig.json",
    "compilerOptions": {
        "isolatedModules": false,
        "target": "es5",
        "lib": ["es5", "dom"],
        "types": [
            "node",
            "cypress",
            "@types/testing-library__cypress",
            "cypress-react-selector"
        ]
    },
    "include": ["../node_modules/cypress", "**/*.ts"]
}
```

The test:
```
    it('Go to magazine page', () => {
        cy.visit('/magazine/test-user-magazine');
        cy.get('h1').should('contain', 'test user magazine');
        cy.get('h2').should('contain', 'test user magazine');
        cy.get('p').should('contain', 'test user magazine description');

        //log in and go to edit page
        cy.login();
        cy.get('a[id="fab"]').click();

        cy.url().should('contain', '/magazine/test-user-magazine/edit');

        cy.waitForReact();
        //change magazine article switch to approved
        cy.get('input[id=defaultAutoAccept]').click({ force: true });
        cy.getReact('MagazineEditComponent')
            .getCurrentState()
            .should('have.property', 'magazine')
            .should('have.property', 'articlesApprovedByDefault')
            .and('eq', true);
    });
 ```
 
 `cy.login()` does log in by request and its command writen by us.
 
Is there some issues with versioning here? Maybe there was breaking change in cypress update?

**Comment by abhinaba-ghosh** [comment]
Hi @Naethaniel , Recently this library got an upgrade on reading `resq` module internally. Now, we make use of `require.resolve` to read the module data [refer] (https://github.com/abhinaba-ghosh/cypress-react-selector/blob/04542b9a528406c596eaba7fe13326457019c00a/src/resqInjector.js#L9). Can you check if this causing the problem or not.

**Comment by Naethaniel** [comment]
Not sure if thats what you meant but here is the screenshot from cypress directly:

![image](https://user-images.githubusercontent.com/16317579/104176656-e6494d80-5407-11eb-86ad-f02958240669.png)

and it looks like it is pointing to that line.

**Comment by abhinaba-ghosh** [comment]
Yep, so I will update this check in the script in the new version. Keep an eye.

**Comment by Naethaniel** [comment]
Awesome, thanks!

**Comment by abhinaba-ghosh** [comment]
Hi @Naethaniel , This is addressed in `v2.3.0`. Feel free to re-open the ticket if the issue still appears. 

If this library helps your development journey and want to get the improvement going, feel free to [sponsor](https://www.buymeacoffee.com/abhinabaghosh) the project 😄

**Comment by KerickHowlett** [comment]
This issue should be reopened, because I'm encountering the same problem with what I'm working on in my Nrwl/Nx monorepo.

![image](https://user-images.githubusercontent.com/8661181/122499611-33d17400-cfbf-11eb-84bc-497ba9058c5a.png)

To get it to work, I have to hard code the absolute file path, which obviously isn't a valid option.
